src/OpeningHoursAlgorithms.h

```diff
@@ -30,27 +30,30 @@
 
 QVariantMap mergeElements(const QVariantList& openingHours,
                           const unsigned from,
-                          const unsigned until)
+                          const unsigned until,
+                          const int todayIndex)
 {
+    QVariantMap result;
+
     if (from == until)
     {
-        return openingHours[from].toMap();
+        result = openingHours[from].toMap();
     }
     else
     {
-        QVariantMap result;
-
         const auto& fromDay  = openingHours[from]. toMap()["day"].toString();
```

Comment:
OpeningHoursAlgorithms.h:44: warning: implicit conversion changes signedness: 'const unsigned int' to 'int'